packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/CryptoSelection/CryptoSelector/CryptoItem/index.tsx

```diff
@@ -19,7 +20,8 @@ class CryptoItem extends PureComponent<Props> {
 }
 
 const mapStateToProps = (state: RootState, ownProps: OwnProps) => ({
-  data: getData(state, ownProps)
+  data: getData(state, ownProps),
+  actionType: selectors.components.simpleBuy.getActionType(state)
```

Comment:
I reused CryptoItem in the EnterAmount step, does it still make sense to reuse it? Should we create a different component for that?

Reply:
they looks similar still, we can see what fits best for us.